db/changelog/v-1/11-changeset-test-agent-insert-data.xml

```diff
@@ -29,6 +29,19 @@
             <column name="status" value="RUNNING" />
             <column name="test_suite_ids" value="1, 2, 3" />
             <column name="resources_root_path" value="resources_root_path" />
+            <column name="offset" value="0" />
+            <column name="execution_limit" value="20" />
+        </insert>
+        <insert dbms="mysql" tableName="execution">
+            <column name="id" value="2" />
+            <column name="project_id" value="2" />
+            <column name="start_time" value="2021-01-01 00:00:00" />
+            <column name="end_time" value="2021-02-01 00:00:00" />
```

Comment:
Can a running execution have end_tie?

Reply:
As we are not considering end time in running state, the column can not be null